sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/Document.java

```diff
@@ -28,6 +28,8 @@
 
     private transient DocumentError documentError;
 
+    private transient boolean mIsFromCache;
```

Comment:
when the getter is "isFromCache" then the backing variable must not contain the is, just `mFromCache`